### PR TITLE
fix: always get the executor's homedir

### PR DESCRIPTION
**Pull request by ngot**
##### Checklist

- [ x ] `npm test` passes
- [ x ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ x ] commit message follows commit guidelines

##### Description of change

总是获取当前执行者的用户目录，保证不会权限错乱。当前执行者用户的判断由启动脚本来判断保证。
错乱的场景：`sudo -u admin node app.js`，此时`id -nu`返回的是`admin`。但是`process.env.HOME`并不是admin

### CODE REVIEW

**Comment by fengmk2** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "is-type-of": "^1.0.0",
     "koa": "^1.2.1",
     "koa-router": "^5.4.0",
+    "node-homedir": "^0.0.1",
```
Comment (fengmk2) [inline package.json]: 以后发模块，先发 1.0.0

**Comment by fengmk2** [comment]
0.3.0

File: package.json (hunk 1 above)
Comment (ngot) [inline package.json]: 收到

**Comment by popomore** [comment]
这个的 Windows 用例挂了，没看？

**Pull request merged**